Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstoryStreamingAPI/GoogleChronicleBackstoryStreamingAPI.py

```diff
@@ -133,6 +134,10 @@ def __init__(self, params: dict[str, Any], proxy, disable_ssl, use_v1_alpha: boo
         # V1 Alpha API parameters
         self.project_id = service_account_credential.get("project_id", "")
         self.project_instance_id = params.get("project_instance_id", "")
+        project_number = params.get("project_number", "")
```

Comment:
```suggestion
        project_number = params.get("project_number")
```
no need to add "" since you're checking in the next line